python_modules/dagster/dagster/_core/definitions/load_assets_from_modules.py

```diff
@@ -313,3 +308,21 @@ def asset2(asset1):
             )
         )
     return result_assets
+
+
+def with_group(
+    assets_defs: Sequence[AssetsDefinition], group_name: Optional[str]
+) -> Sequence[AssetsDefinition]:
+    """
+    Given a list of assets, groups them under the group_name.
+    """
+    group_name = check.opt_str_param(group_name, "group_name")
+    if not group_name:
+        return list(assets_defs)
```

Comment:
nit: input type is sequence, as is output type, so can probably just return assets_defs here, no?